sdk/react/package.json

```diff
@@ -17,6 +17,8 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.11",
-    "hoist-non-react-statics": "^3.3.2"
+    "hoist-non-react-statics": "^3.3.2",
+    "react-native-device-info": "^8.7.0",
```

Comment:
for this package, we need to `import DeviceInfo from 'react-native-device-info'` and set `global.DeviceInfo = DeviceInfo` for RN according to our doc https://docs.devcycle.com/docs/sdk/client-side-sdks/react-native#installation. Can this also be done in our sdk? Or it will affect the react sdk so we shouldn't add it?

Reply:
yeah, we don't want it for React SDK.. so while we will install it, we won't set the `global.DeviceInfo` attribute